resources/views/layouts/default/auth/register.blade.php

```diff
@@ -133,8 +152,7 @@
             <tr>
               <td>
                 <div class="input-group form-group-no-border">
-                  {{ Form::hidden('opt_in', 0, false) }}
-                  {{ Form::checkbox('opt_in', 1, null) }}
+                  <input type="checkbox" name="opt_in" id="opt_in" />
```

Comment:
```html
<input type="hidden" name="opt_in" id="opt_in" value="0">
<input type="checkbox" name="opt_in" id="opt_in" value="1">
```

Reason: When a checkbox is not check it does not get sent by the form, so to reset a selection, there needs to be a hidden field just before the main checkbox.

Ref: HTML Forms, Checkbox, Sending Data

This should be done for all checkboxes specially where editing is possible (or the backend code of these forms should be updated, to check if the checkbox value is received or not, then setting 0 / false etc. as the default value)

Reply:
Thanks